po/zh_TW.po

```diff
@@ -0,0 +1,1173 @@
+msgstr "使用者密碼"
+
+#: ../data/ui/join_panel.glade:309
+msgid "Users"
+msgstr "使用者"
+
+#: ../src/sui/sui_user_list.c:212
+#, c-format
+msgid ""
+"Users: %d, <span color=\"#157915\">%d@</span>,<span color=\"#856117\">%d%%</"
+"span>,<span color=\"#451984\">%d+</span>"
+msgstr "使用者：%d，<span color=\"#157915\">%d@</span>，<span color=\"#856117\">%d%%</span>，<span color=\"#451984\">%d+</span>"
+
+#: ../src/core/app_irc_event.c:725
+#, c-format
+msgid "You are kicked by %1$s"
+msgstr "你被 %1$s 踢出"
+
+#: ../src/core/app_irc_event.c:722
+#, c-format
+msgid "You are kicked by %1$s: %2$s"
+msgstr "你被 %1$s 踢出：%2$s"
+
+#: ../src/core/app_irc_event.c:502
+#, c-format
+msgid "You have joined the channel"
+msgstr "你已加入頻道"
+
+#: ../src/core/login_config.c:52
+#, c-format
+msgid "You need to set the field %2$s in order to use login method %1$s"
+msgstr "你需要設定欄位 %2$s 以使用登入方法 %1$s"
+
+#: ../src/sui/sui_app.c:132
+msgid "[URL…]"
+msgstr "[URL…]"
+
+#: ../data/ui/app.glade:22
+msgid "_About"
+msgstr "關於(_A)"
+
+#: ../data/ui/connect_panel.glade:169
+msgid "_Address:"
+msgstr "地址(_A)："
+
+#: ../data/ui/config_panel.glade:217
+msgid "_Apply"
+msgstr "套用(_A)"
+
+#: ../data/ui/config_panel.glade:203 ../data/ui/prefs_dialog.glade:29
+msgid "_Cancel"
+msgstr "取消(_C)"
+
+#: ../data/ui/nick_menu.glade:41
+msgid "_Chat"
+msgstr "聊天(_C)"
+
+#: ../data/ui/buffer_menu.glade:8
+msgid "_Close"
+msgstr "關閉(_C)"
+
+#: ../data/ui/buffer_menu.glade:14
+msgid "_Disconnect"
+msgstr "中斷連線(_D)"
+
+#: ../data/ui/app.glade:37
+msgid "_Exit"
+msgstr "退出(_E)"
+
+#: ../data/ui/nick_menu.glade:23
+msgid "_Ignore"
+msgstr "忽略(_I)"
+
+#: ../data/ui/nick_menu.glade:32
+msgid "_Kick"
+msgstr "踢出(_K)"
+
+#: ../data/ui/buffer_menu.glade:26
+msgid "_Leave"
+msgstr "離開(_L)"
+
+#: ../data/ui/connect_panel.glade:450
+msgid "_Login method:"
+msgstr "登入方法(_L)："
+
+#: ../data/ui/connect_panel.glade:276
+msgid "_Nickname:"
+msgstr "暱稱(_N)："
+
+#: ../data/ui/config_panel.glade:231 ../data/ui/prefs_dialog.glade:43
+msgid "_OK"
+msgstr "確定(_O)"
+
+#: ../data/ui/connect_panel.glade:153
+msgid "_Password:"
+msgstr "密碼(_P)："
+
+#: ../data/ui/app.glade:13 ../data/ui/app.glade:55
+msgid "_Preferences"
+msgstr "偏好設定(_P)"
+
+#: ../data/ui/url_previewer.glade:26
+msgid "_Preview"
+msgstr "預覽(_P)"
+
+#: ../data/ui/buffer_menu.glade:32
+msgid "_Quit"
+msgstr "退出(_Q)"
+
+#: ../data/ui/buffer_menu.glade:20
+msgid "_Reconnect"
+msgstr "重新連線(_R)"
+
+#: ../data/ui/prefs_dialog.glade:88
+msgid "_Reload"
+msgstr "重新載入(_R)"
+
+#: ../data/ui/connect_panel.glade:185
+msgid "_Server:"
+msgstr "伺服器(_S)："
+
+#: ../data/ui/nick_menu.glade:14
+msgid "_Whois"
+msgstr "_Whois"
+
+#: ../src/render/mention_renderer.c:85
+#, c-format
+msgid "g_regex_new() failed: %1$s"
+msgstr "g_regex_new() 失敗：%1$s"
+
+#: ../src/core/app_irc_event.c:577 ../src/core/app_irc_event.c:652
+#, c-format
+msgid "mode %1$s %2$s by %3$s"
+msgstr "模式 %1$s %2$s 由 %3$s"
+
+#: ../src/config/reader.c:331
+#, c-format
+msgid "server-list[%1$d] doesn't have a name"
```

Comment:
The word `server-list` is the name of a [configuration item](https://srain.silverrainz.me/config.html#all-configurable-items),  it would be better to keep it untranslated.

Reply:
Thanks for your attention to detail, @SilverRainZ!